To: vim-dev@vim.org
Subject: Patch 7.3.003
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.1.003
Problem:    Crash with specific BufWritePost autocmd. (Peter Odding)
Solution:   Don't free the quickfix title twice. (Lech Lorens)
Files:	    src/quickfix.c


*** ../vim-7.3.002/src/quickfix.c	2010-08-15 21:57:26.000000000 +0200
--- src/quickfix.c	2010-08-16 21:15:44.000000000 +0200
***************
*** 2125,2130 ****
--- 2125,2131 ----
  	--qi->qf_lists[idx].qf_count;
      }
      vim_free(qi->qf_lists[idx].qf_title);
+     qi->qf_lists[idx].qf_title = NULL;
  }
  
  /*
*** ../vim-7.3.002/src/version.c	2010-08-16 22:33:55.000000000 +0200
--- src/version.c	2010-08-17 20:23:05.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     3,
  /**/

-- 
From "know your smileys":
 :-X	My lips are sealed

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
